content/docs/100-get-started/200-build-an-app.mdx

```diff
@@ -6,3 +6,20 @@ excerpt: A quick tutorial that covers the basics of building your first app.
 Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
 
 <Callout>Sample callout</Callout>
+
+## My Table
+
+<OptionsTable>
+  <OptionTitle>
+    ### `My Title` 
+  </OptionTitle>
```

Comment:
Had to save this without formatting changes. For some reason when the text is wrapped on one line, it renders differently.

Here is the formatted code:
```
<OptionsTable>
  <OptionTitle>### `My Title`</OptionTitle>
  ...
</OptionsTable>
```
Here is how that renders:
<img width="581" alt="image" src="https://user-images.githubusercontent.com/1712435/174956654-60e10d69-08a5-42d6-9aa4-96d3e5fbfbe7.png">